packages/manager/modules/pci/src/projects/project/instances/instance/instance.controller.js

```diff
@@ -1,3 +1,5 @@
+import { buildURL } from '@ovh-ux/ufrontend/url-builder';
+
```

Comment:
Move all code related to "up-sell cross-sell" to a separate component under instances.  This way its easy to remove or move the code. Refer to https://github.com/ovh/manager/pull/4205 for example

Reply:
done